third_party/packages/cupertino_icons/test/cupertino_icons_test.dart

```diff
@@ -0,0 +1,34 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+/// This test file is primarily here to serve as a source for code excerpts.
```

Comment:
Nit: please add a blank line above this to make it easier to tell apart from the boilerplate header.

Reply:
I pushed this since it was trivial to do in the web UI.